crates/plugins/mediator-coordination/src/forward/routing.rs

```diff
@@ -0,0 +1,225 @@
+use axum::response::{IntoResponse, Response};
+
+use didcomm::{AttachmentData, Message};
+use hyper::StatusCode;
+use mongodb::bson::doc;
+use serde_json::{json, Value};
+
+use crate::{
+
+    model::stateful::entity::RoutedMessage, web::{error::MediationError, AppState, AppStateRepository}
+};
+
+/// mediator receives messages of type forward then it unpacks the messages and stores it for pickup
+/// the unpacked message is then repacked for further transmission.
```

Comment:
The description is not consistent with the actual implementation of the function.